GenerateInfFile.py

```diff
@@ -60,17 +83,27 @@ def getDescription():
     print("Enter multiline ad description. Type 'EOF' on a new line to finish.")
     while True:
         line = input()
-        if line == "EOF":
+        if line.upper() == "EOF":
             break
         contents.append(line)
     return "\\n".join(contents)
 
 
-categoryMap = pickCategory()
+print("****************************************************************")
+print("* Creating the myAd.inf file. Please answer all the questions. *")
+print("****************************************************************\n")
+
+print("Your ad must be submitted in a specific category.")
+
+while True:  # user is able to restart the category picker if anithing goes wrong
```

Comment:
It is because I'm using the returned value to decide if the function should be restarted. But maybe I could turn this logic into a nested function inside pickCategory?